rpc/src/rpc.rs

```diff
@@ -161,6 +161,7 @@ pub struct JsonRpcConfig {
     pub obsolete_v1_7_api: bool,
     pub rpc_scan_and_fix_roots: bool,
     pub max_request_body_size: Option<usize>,
+    pub rpc_prometheus_metrics: bool,
```

Comment:
Maybe a bit more obvious:

```suggestion
    pub rpc_enable_prometheus_metrics: bool,
```

(I see it’s already called `_enable_` in other places, I think it makes sense to use it here too.)